bindings/wasm/lib/bindings.ts

```diff
@@ -20,6 +21,15 @@ const callUtilsMethod = (method: __UtilsMethods__): any => {
     }
 };
 
+const getSecretManagerFromWallet = (
+    method: WalletMethodHandler,
+): SecretManagerMethodHandler => {
+    // TODO figure out why this one is extensible but client isnt
```

Comment:
TODO

Reply:
yea i cannot figure this one out unfortunately, should i make a ticket or do we accept this solution? :)

Reply:
what is the problem exactly?

Reply:
SecretManagerMethodHandler constructor checks the type it receives, if its extensible, its a config/serialized  info. if not, its a rust object, and its ready for use. However, for some reason the SecretManagerMethodHandler we return in wasm is diffferent from the nodejs one, and is extensible, causing it to crash. (trying to deserialize a rust pointer)
Thats why here i intercept the wasm method and just force "not extensible". it works :D

Reply:
(probably because nodejs returns Arc<>, but we cant do that in wasm as its not implementing some trait (i forgot which, some wasm stuff)